stable/admin/templates/statefulset.yaml

```diff
@@ -7,6 +7,9 @@ metadata:
     release: {{ .Release.Name | quote }}
     group: nuodb
     domain: {{ .Values.admin.domain }}
+    {{- if (eq (default "cluster0" .Values.cloud.cluster.name) (default "cluster0" .Values.cloud.cluster.entrypointName)) }}
+    bootstrapServers: {{ default 0 .Values.admin.bootstrapServers | quote }}
+    {{- end }}
```

Comment:
why not inject this value into the startup call?

Are labels mutable? Can someone upgrading from 2.3.1 use this without recreating their admin statefulset?

Reply:
The label is not mutable, as I mention in the comment. It can only be defined when the domain is bootstrapped for the very first time, because the initialMembership has to be defined consistently among all servers in the initialMembership (to prevent disjoint leaders).

Reply:
The labels are not mutable as I mention in the comment in the values file.

The default value is 0 to prevent from breaking Helm chart upgrade, but I'm on the fence about whether to define the default as something else and force customers upgrading to remember to explicitly set `bootstrapServers=0`.

Reply:
So, if we specified the default as 1. Someone upgrading from 2.3.1 would have to override it to 0 to make it work? That sounds like something that can be documented. Also, if you have the value set to either 0 or 1, it basically means that -0 is the bootstrap id, correct?

Reply:
We do not define a bootstrap server at all currently. The fact that admin-0 ends up being the server that all others join through is only enforced by the fact that it is `NUODB_DOMAIN_ENTRYPOINT`. If `NUODB_DOMAIN_ENTRYPOINT`, then there would be nothing forcing servers to join through `admin-0` (I have another branch from a while ago that changes `NUODB_DOMAIN_ENTRYPOINT` to the load-balancer, but there were other problems that blocked it, which should be addressed now).

Reply:
I did not know that! I always assumed that we do use bootstrap of -0.